project/profiles/capacity4more/modules/c4m/user/c4m_user_og/c4m_user_og.module

```diff
@@ -530,15 +539,25 @@ function c4m_user_og_group_actions_cta($group) {
   $items = array();
   // CTAs for the group owner: Manage group and number or new membership
   // requests.
-  if (c4m_og_user_is_owner('node', $group->nid)) {
+  if ($user->uid == $group->uid) {
```

Comment:
This way, it only works for Group Owners, but the description on CFM-600 says "GO/GA", so I expect it should work for Group Administrators too...